others/sub_sequences.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 by Contributors
```

Comment:
use doxygen comment style
```
/**
 *
 *
 */
```
also write proper documentation of work whatever you have done